packages/tree_hash/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "tree_hash"
+version = "0.8.0"
+edition = "2021"
+description = "Efficient Merkle-hashing as used in Ethereum consensus"
+license = "Apache-2.0"
+readme = "../README.md"
+repository = "https://github.com/sigp/tree_hash"
```

Comment:
Why is this copied here? was there no other way to do this?

Reply:
It is forked. It uses a library that has some random features that causes issues with CosmWasm :/